src/main/java/io/github/protonmc/proton/mixin/world/DefaultBiomeFeaturesMixin.java

```diff
@@ -0,0 +1,24 @@
+package io.github.protonmc.proton.mixin.world;
+
+import io.github.protonmc.proton.module.ModuleManager;
+import io.github.protonmc.proton.module.world.ClayModule;
+import net.minecraft.world.biome.GenerationSettings;
+import net.minecraft.world.gen.GenerationStep;
+import net.minecraft.world.gen.feature.DefaultBiomeFeatures;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(DefaultBiomeFeatures.class)
+public class DefaultBiomeFeaturesMixin {
+
+    @Inject(at = @At("HEAD"), method = "addMineables")
```

Comment:
I always prefer injections like this being at `RETURN`

Reply:
I do not agree with this, As some other mod could potentially mess stuff up.